src/Tools/dotnet-monitor/CollectionRules/Options/Actions/GetEnvironmentVariableOptions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options.Actions
+{
+    /// <summary>
+    /// Options for the <see cref="CollectionRules.Actions.GetEnvironmentVariableAction"/> action.
+    /// </summary>
+    [DebuggerDisplay("GetEnvironmentVariable")]
+    internal class GetEnvironmentVariableOptions
+    {
+        [Display(
+            ResourceType = typeof(OptionsDisplayStrings),
+            Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_GetEnvironmentVariableOptions_Name))]
+        [Required]
+        public string Name { get; set; }
```

Comment:
Did you want to add some validation over the name? I don't believe every character is allowed for environment names.

Reply:
I looked into it and didn't want to over-validate because Windows and Linux have different characters that are allowed in names.